pyproject.toml

```diff
@@ -44,7 +42,11 @@ dependencies = [
     "edt>=2.3.1",
     "astropy>=5.2",
     "rich",
-    "boto3"
+    "boto3",
+    "bioio",
+    "bioio-czi",
+    "bioio-ome-tiff",
+    "bioio-tifffile"
 ]
 requires-python = ">=3.8,<3.11"
```

Comment:
Can you change this to >=3.9?

Reply:
Done! [9d3f60](https://github.com/AllenCellModeling/cyto-dl/pull/376/commits/1b651d95ad40b128987a63e0f620c7981c9d3f60)